lib/backends/sdc/utils.js

```diff
@@ -942,16 +946,26 @@ function compressPorts(ports) {
     return ranges;
 }
 
+/*
+ * Escapes strings for ldap filters.
+ * See: pfmooney/node-ldap-filter/lib/substr_filter.js#L12-L15
+ */
+function ldapEscape(str) {
+    /* JSSTYLED */
+    return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
```

Comment:
Not without a new ldap-filter release, and maybe breaking its intended usage a bit. @pfmooney and I thought this was the best compromise.

There is a longer-term plan here, see [NAPI-367](https://smartos.org/bugview/NAPI-367), that should remove the requirement for any ldap filters on input, but right now this is required as a workaround (I'll note that too in next rev).